lib/src/components/NxThreatCounter/NxThreatCounter.tsx

```diff
@@ -12,8 +12,8 @@ export { Props } from './types';
 
 import './NxThreatCounter.scss';
 
-export default function NxThreatCounter({
-  criticalCount, severeCount, moderateCount, lowCount, noneCount, layout, className, ...attrs }: Props) {
+export default function NxThreatCounter({ criticalCount, severeCount, moderateCount, lowCount,
+  noneCount, unspecifiedCount, layout, className, ...attrs }: Props) {
```

Comment:
Janky formatting

Reply:
> My approach when the number of props gets too long is to do the destructing in a local variable assignment rather than directly in the function parameters

This has been implemented: 8aa87f9724ea0f141e76ac404f4db2a535155bac